src/server.py

```diff
@@ -556,6 +559,46 @@ def run_dailyjob_query():
     return {'dailyjobs': output}
 
 
+# It's a temp endpoint to make database migration from alermanager server to heroku more easier
+@app.route("/data/dump/")
+@json_response
+def run_old_job_dump():
+    limit = request.args.get('limit')
+    offset = request.args.get('offset')
+    date_format = "%04d-%02d-%02d %02d:%02d:%02d"
+    db = create_db_connnection()
+    cursor = db.cursor()
+    query = "select slave, result, build_system_type, duration, platform, buildtype, " \
+            "testtype, bugid, branch, revision, date, failure_classification, failures " \
+            "from testjobs LIMIT %s OFFSET %s" % (limit, offset)
```

Comment:
is there any concern that the query will return things in a different order making it possible to miss data on limit/offset?  

lastly how do we plan to use this, set limit to a value and keep incrementing offset until results is empty:
limit = 5000
offset = 0
results = getdump(limit, offset)
while (len(results) > 0):
    insertDataToHeroku(results)
    insertDataToHerokuStaging(results)
    offset += limit
    results = getdump(limit, offset)